baselines/jft/deterministic.py

```diff
@@ -28,15 +28,13 @@
 from clu import periodic_actions
 from clu import preprocess_spec
 import flax
-import flax.jax_utils as flax_utils
 import jax
 import jax.numpy as jnp
-from ml_collections.config_flags import config_flags
```

Comment:
This line seems to have broken the tests. 

```
AttributeError: module 'ml_collections' has no attribute 'config_flags'
```

https://github.com/google/uncertainty-baselines/runs/4944001665?check_suite_focus=true

Reply:
thanks for catching this. fixed in https://github.com/google/uncertainty-baselines/pull/711